src/content/docs/en/guides/cms/cloudcannon.mdx

```diff
@@ -9,27 +9,329 @@ service: CloudCannon
+
+<Steps>
+
+1. In your CloudCannon Organization Home page, create a new Site.
+2. Authenticate your Git provider and select the Astro repository you want to connect to.
+3. Choose a name for your Site, then CloudCannon will create the Site and start syncing your files.
+4. Follow CloudCannon's guided tasks in your Site dashboard for completing your Site setup, including creating a CloudCannon configuration file (`cloudcannnon.config.yml`)
+5. Save your configuration file to commit it with your CloudCannon preferences to your Git repository.
+
+</Steps>
+
+You can now explore your Site Dashboard to see your Astro files and edit them with the Content Editor.
+
+You may also want to take advantage of some CloudCannon features such as [organizing your files into collections](#cloudcannon-collections-and-schemas), [creating CloudCannon schemas](#create-a-cloudcannon-schema-for-a-collection), and setting up your project for [visual editing](#configure-visual-editing).
+
+For more detailed instructions, see [CloudCannon's Getting Started Guide](https://cloudcannon.com/documentation/guides/getting-started-with-cloudcannon/).
+
+## CloudCannon collections and schemas
+
+If you use [Astro's content collections](/en/guides/content-collections/), then you will be familiar with CloudCannon's concepts of collections (used for organization/navigation in your Site Dashboard) and schemas (used to define the format of new content entries).
+
+Your CloudCannon Site Dashboard allows you to organize your Astro project's pages and content into collections: groups of related files with a similar format. This allows you to see similar types of content together for ease of editing and makes your content files easy to navigate, sort, and filter. You can set collection-level configuration features such as a folder path and output URL, as well as specify which editing views are available to editors working on files in this collection. This can be done through your Site Dashboard or by editing your CloudCannon configuration file directly in your Astro project.
+
+By default, CloudCannon will automatically try to detect appropriate collections (e.g. blog posts, individual pages) when you first create your CloudCannon configuration file by searching your repository files for folders of similar content. You can also [manually group files into CloudCannon collections](https://cloudcannon.com/documentation/guides/getting-started-with-cloudcannon/group-files-into-collections/) to customize your view.
+
+You can define blank entry templates for your content types by adding one or more [CloudCannon schemas](https://cloudcannon.com/documentation/articles/what-is-a-schema/). These allow you to quickly create new content entries (e.g. blog posts, newsletters, author pages) through your Site Dashboard that will have the proper default fields, ready to be filled in.
```

Comment:
I feel like this part is mostly covered in the "Create a CloudCannon schema for a collection" section and introduce some extra complexity "blank entry template" too early while this is properly explained later. Any thoughts?